Library/Homebrew/extend/os/cygwin/cleaner.rb

```diff
@@ -0,0 +1,7 @@
+class Cleaner
+  private
+
+  def executable_path?(path)
+    path.extname == ".exe" || path.text_executable?
```

Comment:
I worked on an extended path for PE format executables, like the Linux one for ELF. I know the magic bytes. There's an "MZ" at byte 0x0 and a "PE" at byte 0x80, but for some reason my code didn't work. If you think this needs that, I can work on it.